.circleci/build.sh

```diff
@@ -49,10 +49,11 @@ install_fmt() {
     popd
 }
 
-GLOW_DEPS="libpng-dev libgoogle-glog-dev libboost-all-dev libdouble-conversion-dev libgflags-dev libjemalloc-dev libpthread-stubs0-dev libevent-dev libssl-dev"
+GLOW_DEPS="libpng-dev libgoogle-glog-dev libboost-all-dev libdouble-conversion-dev libgflags-dev libjemalloc-dev libpthread-stubs0-dev libevent-dev libssl-dev g++7"
+
+sudo -E apt-add-repository -y "ppa:ubuntu-toolchain-r/test"
```

Comment:
I just noticed that if you do 
```
sudo update-alternatives ...
```
as listed in https://github.com/pytorch/glow#ubuntu section. It will also work.